src/components/FacilityInfo/index.tsx

```diff
@@ -87,7 +87,7 @@ export const FacilityInfo: FC<FacilityInfoType> = ({
             {facility.fields.Schlagworte && (
               <TagItem
                 question="Schlagworte"
-                answers={facility.fields.Schlagworte.split(';')}
+                answers={facility.fields.Schlagworte.map((n) => `${n}`)}
```

Comment:
The tags are now displayed like this:

![Screenshot 2022-09-23 at 17 14 31](https://user-images.githubusercontent.com/15640196/191994523-18f109a8-1151-4d32-ac99-fd17085486b8.png)

Is this expected to be broken at this stage?

Reply:
Oh yes I need to fix this later. TBH I didn't even look into the map and the sidebar because we need to take care of those individually anyway. But you're right. We need to map the label IDs to the label names. :)

Reply:
True, that's a separate task!